presto-docs/src/main/sphinx/admin/graceful-shutdown.rst

```diff
@@ -0,0 +1,34 @@
+=================
+Graceful Shutdown
+=================
+
+Presto has a graceful shutdown API that can be used exclusively on workers in
+order to ensure that they terminate without affecting running queries, given a sufficient
+grace period.
+
+You can invoke the API like this:
+
+.. code-block:: bash
+
+    curl -v -X PUT -d '"SHUTTING_DOWN"' -H "Content-type: application/json" \
+        http://worker:8081/v1/info/state
```

Comment:
I think it'll be nice to add the log message that the workers print when the request is received successfully so that people who are trying this out can make sure it's working properly for them.

https://github.com/prestosql/presto/blob/master/presto-main/src/main/java/io/prestosql/server/GracefulShutdownHandler.java#L75

Reply:
Done.